src/mlpack/core/data/load_impl.hpp

```diff
@@ -393,6 +393,37 @@ bool Load(const std::string& filename,
   return true;
 }
 
+// For loading data into sparse matrix
+template <typename eT>
+bool Load(const std::string& filename,
+          arma::SpMat<eT>& matrix,
+          const bool fatal,
+          const bool transpose)
+{
+  bool success;
+  Timer::Start("loading_data");
+  success = matrix.load(filename, arma::coord_ascii);
+
+  if (transpose)
+  {
+    inplace_transpose(matrix);
```

Comment:
Shouldn't we handle the output from `inplace_transpose()`?  (I suppose this would also apply to the other overloads of `Load()`.)

Reply:
@rcurtin  
Thanks for the review!

Could you help me understand what is meant by "handling the output from `inplace_transpose()`" ?

Reply:
The function returns a `bool`---we're not handling that in any way.  I think that we should handle it. :+1: